lambda_main.py

```diff
@@ -321,12 +326,15 @@ def upload_handler(event, context):
 
     """
     try:
+        plog(f"Event: {event}")
         # bucket = event['Records'][0]['s3']['bucket']['name']
         key = event["Records"][0]["s3"]["object"]["key"]
         path = key.split("/")[-1]
+        plog(f"Key: {key}")
+        plog(f"Path: {path}")
         transfer_s3_path_to_remarkable(path)
     except Exception as e:
-        return {"statusCode": 500, "body": "Failure occurred!"}
+        return {"statusCode": 500, "body": f"Failure occurred: {e}"}
```

Comment:
This was deliberate, as sending back a traceback may have more information than is useful or good to share (i.e. if a db dump or excerpt is included in the traceback)

Reply:
Are cloudwatch logs public?

Reply:
I've reverted it regardless.

Reply:
sure hope not! but good practice to not expose these to the caller, no matter who it is